src/Service/Models/GraphQLFilterParsers.cs

```diff
@@ -182,8 +181,21 @@ public GQLFilterParser(ISqlMetadataProvider metadataProvider)
                         }
                         else
                         {
+                            // This path will never get called for sql since the primary key will always required
+                            // This path will only be exercised for Cosmos
```

Comment:
nit: Cosmos_nosql

Reply:
Updated

Reply:
sorry didnt realize it is `cosmosdb_nosql`